book_project/book_api.py

```diff
@@ -0,0 +1,194 @@
+import uvicorn,json
+from fastapi import FastAPI
+import mysql.connector
+from pydantic import BaseModel
+from fastapi.responses import JSONResponse
+
+app = FastAPI()
+
+def db_connect():
+    return mysql.connector.connect(
+        host='3.7.198.191',
```

Comment:
Credentials should not be part of the code. Ideal to have these as env variables